llvm/test/Transforms/LoopVectorize/AArch64/scatter-cost.ll

```diff
@@ -0,0 +1,8 @@
+; RUN: opt -mtriple aarch64 -mcpu=neoverse-v2 -passes="print<cost-model>" -disable-output < %s 2>&1 | FileCheck %s
+; CHECK: Cost Model: Found an estimated cost of 52 for instruction: call void @llvm.masked.scatter.nxv4f32
```

Comment:
Can you add a run line without V2, to show the difference between the two.

Reply:
Added a generic case for which cost model deduces "Invalid". I hope that is valuable to have.

Reply:
Oh that's probably because it's scalable with no sve support. Can you add -mattr=+sve2 to make them comparable?

Reply:
Ok, addressed in the latest commit.